doc/UserGuide/BuildingRunningTesting/InputsOutputs.rst

```diff
@@ -44,41 +42,157 @@ UFS Unified Model Variables
 ITAG
 ----
 
-The file called ``itag`` is a text file that contains the fortran namelist ``&model_inputs`` as
-well as the 2D decomposition specification in ``&nampgb``. It is read in by the ``upp.x`` executable
-from standard input (stdin -- unit 5) and is generated automatically within the UFS application workflow or standalone run 
-script based on user-defined options. It should not be necessary to edit this file. For description purposes,
-the namelists ``&model_inputs`` and ``&nampgb`` (in the ``itag`` file) contain the following lines for FV3:
+The file called ``itag`` is a Fortran namelist file that contains two sections: ``&model_inputs`` and ``&nampgb``. It is read in by the ``upp.x`` executable from standard input (stdin -- unit 5). Most UFS applications generate it automatically based on user-defined application-level options. All UPP namelist choices are described here.
 
 :bolditalic:`&model_inputs`
 
-#. ``fileName``: Name of the FV3 (pressure level) output file to be post-processed.
-
-#. ``IOFORM``: Format of FV3 model output (netcdfpara).
-
-#. ``grib``: Format of UPP output (grib2)
-
-#. ``DateStr``: Forecast valid time (not model start time) in YYYY-MM-DD_HH:00:00 format (the forecast time
-   desired to be post-processed).
-
-#. ``MODELNAME``: Model used (GFS, FV3R --- regional FV3, also known as the :term:`LAM`).
-
-#. ``fileNameFlux``: Name of the FV3 (surface) output file to be post-processed.
-
-#. ``fileNameFlat``: Name of configuration file (``postxconfig-NT.txt``)
+The ``&model_inputs`` section parameterizes choices about the set of model output files that will be used for the UPP 
+
+.. list-table:: *Description of the &model_inputs namelist section.*
+   :widths: 15 40 20 20
+   :header-rows: 1
+
+   * - Variable Name
+     - Description
+     - Data Type
+     - Default Value
+   * - ``datestring``
+     - Time stamp being processed (e.g., 2022-08-02_19:00:00).
+     - character(len=19)
+     - n/a
+   * - ``fileName``
+     - Name of input dynamics file; name of full 3-D model output file.
+     - character(len=256)
+     - n/a
+   * - ``filenameflat``
+     - Input configuration text file defining the requested fields.
+     - character(len=256)
+     - postxconfig-NT.txt
+   * - ``filenameflux``
+     - Name of input physics file; name of 2-D model output file with physics and surface fields.
+     - character(len=256)
+     - n/a
+   * - ``grib``
+     - Grib type (Note that UPP only supports Grib2 currently)
+     - character(5)
+     - grib2
+   * - ``ioform``
+     - Input file format. Choices: binary, binarympiio, binarynemsio, netcdf, netcdfpara
+     - character(len=20)
+     - n/a
+   * - ``modelname``
+     - Model name used by UPP internally (e.g., FV3R for LAM, GFS for GFS, NCAR for WRF)
```

Comment:
@christinaholtNOAA I would suggest update as:
e.g., FV3R for RRFS, 3DRTMA, HAFS; GFS for GFS and GEFS; RAPR for RAP and HRRR; NMM for NAM
** 'NCAR' is not a valid UPP internal model name any more.